whois-api/src/main/java/net/ripe/db/whois/api/rdap/WhoisRdapService.java

```diff
@@ -223,6 +223,18 @@ public Response searchDomains(
         return handleSearch(new String[]{"domain"}, name, request);
     }
 
+    @GET
+    @Produces({MediaType.APPLICATION_JSON, CONTENT_TYPE_RDAP_JSON})
+    @Path("/help")
+    public Response help(@Context final HttpServletRequest request) {
+
+        LOGGER.info("Request: {}", RestServiceHelper.getRequestURI(request));
+
+        return Response.ok(rdapObjectMapper.mapHelp(getRequestUrl(request)))
+                .header("Content-Type", CONTENT_TYPE_RDAP_JSON)
```

Comment:
Can you replace "Content-Type" with javax.ws.rs.core.HttpHeaders.CONTENT_TYPE.

(this is a general cleanup, there are a few existing examples in that class, not just your change).